apps/dashboard/src/app/app-routing.module.ts

```diff
@@ -166,7 +165,10 @@ const routes: Routes = [
               },
               {
                 path: 'processes',
-                component: KnowledgeBoxProcessesComponent,
+                loadChildren: () =>
+                  import('./knowledge-box/knowledge-box-process/knowledge-box-processes.module').then(
```

Comment:
Maybe it would worth renaming this module and corresponding components and routes into `KnowledgeBoxTrainingModule`?

Reply:
yes, sounds better, let me rename it

Reply:
done!